runtime/compiler/control/JITClientCompilationThread.cpp

```diff
@@ -500,6 +500,19 @@ handleServerMessage(JITServer::ClientStream *client, TR_J9VM *fe, JITServer::Mes
          vmInfo._floatInvokeExactThunkHelper = comp->getSymRefTab()->findOrCreateRuntimeHelper(TR_icallVMprJavaSendInvokeExactF, false, false, false)->getMethodAddress();
          vmInfo._doubleInvokeExactThunkHelper = comp->getSymRefTab()->findOrCreateRuntimeHelper(TR_icallVMprJavaSendInvokeExactD, false, false, false)->getMethodAddress();
          vmInfo._interpreterVTableOffset = TR::Compiler->vm.getInterpreterVTableOffset();
+         vmInfo._jlrMethodInvoke = fe->_vmThread->javaVM->jlrMethodInvoke;
```

Comment:
`vmThread` is available at the beginning of this method. Use that instead.

Reply:
Since `javaVM` is used in several places you could store to a local and use that instead.

Reply:
I think you need to acquire VMAccess because you are working with objects which can be changed by GC